satellite/satellitedb/overlaycache.go

```diff
@@ -97,12 +95,13 @@ func (cache *overlaycache) SelectNewStorageNodes(ctx context.Context, count int,
 	nodeType := int(pb.NodeType_STORAGE)
 
 	safeQuery := `
-		WHERE type = ? AND free_bandwidth >= ? AND free_disk >= ?
-		  AND total_audit_count < ? AND audit_success_ratio >= ?
+		WHERE type = ? AND NOT disqualified
```

Comment:
Same comments in this query, alignment, breaking down each column in a new line and optionally having the `NOT disqualified` as the first condition of the `WHERE` clause.